packages/fuels/tests/Forc.toml

```diff
@@ -0,0 +1,74 @@
+[workspace]
```

Comment:
I would move `Forc.toml` one level above.

Reply:
If you do this do not forget tho update the docs

Reply:
ill try :)